lunar/distribution.yaml

```diff
@@ -943,6 +943,28 @@ repositories:
       url: https://github.com/mavlink/mavlink-gbp-release.git
       version: release/lunar/mavlink
     status: maintained
+  mavros:
+    doc:
+      type: git
+      url: https://github.com/mavlink/mavros.git
+      version: master
+    release:
+      packages:
+      - libmavconn
+      - mavros
+      - mavros_extras
+      - mavros_msgs
+      - test_mavros
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/mavlink/mavros-release.git
+      version: 0.19.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Not being part of the mavlink organization I cannot confirm. Did you give the rights to ros-pull-request-builder on your organization so that the PRs can be tested?